pkgs/by-name/vi/vintagestory/package.nix

```diff
@@ -116,4 +119,4 @@ stdenv.mkDerivation rec {
     ];
     mainProgram = "vintagestory";
   };
-}
+})
```

Comment:
I noticed that the meta.platforms is specified as lib.platforms.linux, but it packages the x86_64-linux binary.

Should it be specified as "x86_64" instead?